src/java/org/apache/cassandra/db/marshal/TupleType.java

```diff
@@ -383,7 +383,7 @@ public <V> V pack(List<V> elements, ValueAccessor<V> accessor)
 
     public ByteBuffer pack(ByteBuffer... components)
     {
-        return pack(Arrays.asList(components), ByteBufferAccessor.instance);
+        return pack(Arrays.asList(components));
```

Comment:
when i did the last rebase this change got added else the history got too hard to work with; so rolling back